### PR TITLE
applications: machine_learning: Increase power down timeout

### PR DESCRIPTION
**Pull request by MarekPieta**
Change increases CAF power manager timeout from 10 seconds to 30 seconds to make sure that Bluetooth Central in CI tests will have enough time to establish Bluetooth connection.

Jira: NCSDK-14601

### CODE REVIEW

**Review by jumi-nordic — APPROVED** [review]

**Comment by MarekPieta** [comment]
Added note to release notes changelog.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -107,6 +107,11 @@ nRF9160: Serial LTE Modem
   * Added native TLS support to the HTTPS client.
   * Added the #XCMNG command to support the use of native TLS.
 
+nRF Machine Learning (Edge Impulse)
+-----------------------------------
+
+* Increased value of :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` to 30 seconds on Thingy:53.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:113]:
```suggestion
* Increased the default value of :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` to 30 seconds for Thingy:53.
```

**Review by emob-nordic — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst]: Change a bit to:
`Increased the value of :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` to 30 seconds for Thingy:53.`
The Kconfig option value is set to 30 seconds in application configuration, we do not actually modify default value.

**Review by divipillai — APPROVED** [review]

**Review by pdunaj — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Comment by MarekPieta** [comment]
Force pushed to retrigger CI

**Pull request merged**